frontend/question.html

```diff
@@ -3,64 +3,100 @@
+                        <label for="question-1-answers-C"> Crazy Solid Shapes</label>
+                    </div>
 
-            <div class="quest-1">
-                <li>
+                    <div>
+                        <input type="checkbox" name="question-1-answers" id="question-1-answers-D" value="D" />
+                        <label for="question-1-answers-D"> None of the above</label>
+                    </div>
+                </div>
+            </div>
+        </div>
 
-                    <h3>What do you prefer as a hobby/ what do you do in your free time</h3>
+        <div class="qst-cont">
+            <div class="questions-bx sec">
+                <div class="question-#">
+                    <p>What do you prefer as a hobby/ what do you do in your free time</p>
+                </div>
+                <div class="hr">
                     <hr>
-
+                </div>
+                <div class="options">
                     <div>
-                        <input type="radio" name="question-1-answers" id="question-1-answers-A" value="A" />
-                        <label for="question-1-answers-A">A) Computer Styled Sections </label>
+                        <input type="checkbox" name="question-1-answers" id="question-1-answers-A" value="A" />
+                        <label for="question-1-answers-A"> Computer Styled Sections </label>
                     </div>
 
                     <div>
-                        <input type="radio" name="question-1-answers" id="question-1-answers-B" value="B" />
-                        <label for="question-1-answers-B">B) Cascading Style Sheets</label>
+                        <input type="checkbox" name="question-1-answers" id="question-1-answers-B" value="B" />
+                        <label for="question-1-answers-B"> Cascading Style Sheets</label>
                     </div>
 
                     <div>
-                        <input type="radio" name="question-1-answers" id="question-1-answers-C" value="C" />
-                        <label for="question-1-answers-C">C) Crazy Solid Shapes</label>
+                        <input type="checkbox" name="question-1-answers" id="question-1-answers-C" value="C" />
+                        <label for="question-1-answers-C"> Crazy Solid Shapes</label>
                     </div>
 
                     <div>
-                        <input type="radio" name="question-1-answers" id="question-1-answers-D" value="D" />
-                        <label for="question-1-answers-D">D) None of the above</label>
+                        <input type="checkbox" name="question-1-answers" id="question-1-answers-D" value="D" />
+                        <label for="question-1-answers-D"> None of the above</label>
                     </div>
-
-                </li>
-
+                </div>
             </div>
         </div>
 
-        <div>
-            <div id="button">
-                <button type="button">Back</button>
+        <div class="row">
```

Comment:
so no need for this row here, just use (after making the reviewed changes on `styles.css`)
```
 <p id="inner" align="center">
   <a href="#">Back</a>
   <a href="#">Next</a>
</p>
```

Reply:
Done